renovate.json5

```diff
@@ -0,0 +1,61 @@
+{
+  "extends": ["apollo-open-source", ":semanticCommits"],
+  "enabledManagers": ["npm", "cargo"],
+  "packageRules": [
+    // Bunch up all non-major npm dependencies into a single PR.  In the common case
+    // where the upgrades apply cleanly, this causes less noise and is resolved faster
+    // than starting a bunch of upgrades in parallel for what may turn out to be
+    // a suite of related packages all released at once.
+    //
+    // Since too much in the Rust ecosystem is pre-1.0, we make an exception here.
+    {
+      "matchPackageNames": ["harmonizer"],
+      "groupName": "harmonizer",
+      "groupSlug": "harmonizer"
+    },
+    {
+      "matchCurrentVersion": "< 1.0.0",
+      "separateMinorPatch": true,
+      "matchManagers": ["cargo"],
+      "minor": {
+        "groupName": "cargo pre-1.0 packages",
+        "groupSlug": "cargo-all-pre-1.0"
+      },
+      "patch": {
+        "groupName": "cargo pre-1.0 packages",
+        "groupSlug": "cargo-all-pre-1.0",
+        "automerge": false
```

Comment:
Safe place to start. 😉 

(You can remove this line too, if you never want this.)